src/IB/IBFESurfaceMethod.cpp

```diff
@@ -769,6 +1416,7 @@ IBFESurfaceMethod::interpolateVelocity(const int u_data_idx,
     return;
 } // interpolateVelocity
```

Comment:
It was already a problem, but in `IBFEMethod` we moved the vector communication and projection outside of the loop over the parts. Its necessary to do this in phases so that we don't have severe load imbalances when running with multiple parts and multiple MPI processes.

We will have to fix this in a future PR.